pkg/auth/proxy_test.go

```
@@ -0,0 +1,86 @@
+
+import (
+	"net/http/httptest"
+	"testing"
+)
+
+func TestNewProxyAuthOffOnlyWhenBothEmpty(t *testing.T) {
+	p, err := NewProxyAuth("", nil)
+	if err != nil || p != nil {
+		t.Fatalf("both empty must mean off: %v %v", p, err)
+	}
+	if _, ok := p.Identify(httptest.NewRequest("GET", "/", nil)); ok {
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)